modules/dashboard/test/TestPlan.md

```diff
@@ -1,77 +1,61 @@
-    account approvals is displayed in the My Task panel. This should be the number of entries in the User Account
-    page with the following Selection Filter: Site set to the user's site and Pending Approval set to 'Yes'. The
-    Site displayed will be 'All user sites'. Check that you are taken to that page (with the Selection Filter
-    correctly set) when you click on the task.
-    [Automate Test on Travis_CI]
-16. Verify that a user with 'Violated Scans: View all-sites Violated Scans' permission has a task with the number
-    of violated scans displayed. This is the number of entries on the MRI Violated Scans page. The Site displayed will
-    always be 'All'. Check that clicking on the task takes you to the Violated Scans page.
-    [Automate Test on Travis_CI]
-17. Verify that if a user has the 'View and upload files in Document Repository' or 'Delete files in Document Repository'
+3. Verify that for a user with 'Resolving conflicts' permission the number of data entry conflicts is reported in the
+   My Task panel. If the user also has 'Across all sites access candidates profiles' then the site displayed is
+   'All', otherwise it is set to 'All user sites'. The number of data entry conflicts is the number of
+   entries in the Unresolved tab of the Conflict Resolver page for candidates that do not belong to the DCC site. 
+   Click on this task and check that you go to the Conflict Resolver page.
+   [Automate Test on Travis_CI]
+4. Check that for a user with 'Data Entry' permission, the number of incomplete forms (instruments with Data Entry
+   set to 'In Progress') is displayed in the My Tasks panel. If the user also has 'Across all sites access candidates
+   profiles' then the site displayed is 'All', otherwise it is set to 'All user sites' and only the
+   candidates that belong to the user's site are considered for the computation of the number of incomplete forms.
+   Clicking on this task should take you to the BVL statistics page, with the stats filtered according to the user's
+   site (or without any filter if the user has 'Across all sites access candidates profiles' permission).
+   [Automate Test on Travis_CI]
+5. Verify that if a user has 'Across all sites create and edit users' and 'User management' permissions, the number of pending
+   account approvals is displayed in the My Task panel. This should be the number of entries in the User Account
+   page with the following Selection Filter: Site not set and Pending Approval set to 'Yes'. 
+   Check that you are taken to that page (with the Selection Filter correctly set) when you
+   click on the task.
+   [Automate Test on Travis_CI]
+6. Verify that if a user has 'Data entry' permission, the reports menu is displayed and the number of incomplete
+   forms show in the tasks list.
+   [Automate Test]
+7. Verify that if a user has 'issue_tracker_reporter / issue_tracker_developer' permission,
+   the issue tracker panel should not be found without this permission.
```

Comment:
Theses last 2 lines could be rephrased IMO.

Reply:
ok